Source/Provers/SMTLib/SMTLibInteractiveTheoremProver.cs

```diff
@@ -713,7 +712,14 @@ protected override void Send(string s, bool isCommon)
 
     public override async Task<int> GetRCount()
     {
-      SendThisVC("(get-info :rlimit)");
+      if (options.Solver != SolverKind.Z3) {
+        // Only Z3 currently supports retrieving this value. CVC5
+        // supports setting a limit, but does not appear to support
+        // reporting how much it took to complete a query.
```

Comment:
That's unfortunate! Can we log an issue for it if it doesn't already exist?

Could we say `if (options.Solver == SolverKind.CVC5)` instead (here and in the other spots you changed), so we default to at least attempting this for other solvers, especially new ones?

Reply:
Z3 is the only currently-supported solver that has that feature, I think. Yices2 also doesn't support it.

Reply:
Okay, not the end of the world. If we move towards trying to run all tests on all solvers by default, that will be another mechanism to push us towards hooking up this feature when it is supported.